ui/src/Panels/Producer/Producer.stories.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withKnobs, number } from '@storybook/addon-knobs';
+import { withInfo } from '@storybook/addon-info';
+import { Producer } from './index.js';
+import ProducerReadme from './README.md';
+
+import { producerMockWebsocket } from './Producer.assets.js';
+
+const renderHelper = () => () => {
+  const count = number('Count', 5);
+  return (
+    <Producer
```

Comment:
You will also want to have this as well https://github.com/matthew-chirgwin/kafka-java-vertx-starter/blob/7a2230a4350ae3a61f0b83c87c4c58a75ba46873/ui/src/Panels/Consumer/Consumer.stories.js#L14 when integrating the selected message stuff

Reply:
Also added